application-forum-ui/src/main/resources/ForumCode/ForumSheet.xml

```diff
@@ -54,16 +54,8 @@
 
   #elseif($xcontext.action == 'view')
     #initForum($spaceName)
-    #set ($supportsNS = $services.extension.core.getCoreExtension('org.xwiki.platform:xwiki-platform-model').id.version.compareTo($services.extension.parseVersion('7.2')) &gt;= 0)
-    #if ($supportsNS)
-    ## define some variables to be used with Nested Spaces
-      #set($hierarchyCondition = "doc.space=:forumDoc")
-      #set($hierarchyBinding = $doc.space)
-    #else
-      #set($hierarchyCondition = "doc.parent=:forumDoc")
-      #set($hierarchyBinding = $doc.fullName)
-    #end
-    #set($topics = $services.query.xwql("from doc.object(ForumCode.TopicClass) as topic where $hierarchyCondition").bindValue('forumDoc', $hierarchyBinding).execute())
+    #set($hierarchyCondition = "doc.parent=:forumDoc or doc.space=:forumSpace")
```

Comment:
Would be nice to have some comment for this line. It'll be hard for someone reading it in the future to guess why there's an OR here. Best would be to add a comment explaining why there's this OR. WDYT?